index.js

```diff
@@ -7,6 +7,14 @@ const { flow } = require('./lib/configs/flow')
 const { recommended } = require('./lib/configs/recommended')
 
 module.exports = {
+    prettier: {
+        arrowParens: 'avoid',
```

Comment:
I know that this is the 2.0 big change, but I think we should go with the default value (`always`)  here. It will make the code more consistent and allows adding a second param or types more easily. It's well described: [here](https://prettier.io/docs/en/options.html#arrow-function-parentheses)

Reply:
I don't mind - as long as it is consistent. Besides - we add types to parameters so probably we will have parens present

Reply:
https://github.com/pmedianetwork/eslint-config/pull/86/commits/09a4e685abf9e35d447e0d5f71fad42c05de0c12